src/api/wix/WixToolset.Data/WixOutput.cs

```diff
@@ -73,6 +91,19 @@ public static WixOutput Create(Uri uri, Stream stream)
             return new WixOutput(uri, archive, stream);
         }
 
+        /// <summary>
+        /// Creates a new file structure that can only be written to.
+        /// </summary>
+        /// <param name="uri"></param>
+        /// <param name="stream">Stream to write the file structure to.</param>
+        /// <returns>Newly created <c>WixOutput</c>.</returns>
+        public static WixOutput CreateNew(Uri uri, Stream stream)
```

Comment:
Since we're trying to be minimal on the API surface, let's merge this CreateNew with the CreateNew above.